vlasiator.cpp

```diff
@@ -525,23 +527,9 @@ int main(int argn,char* args[]) {
       phiprof::stop("compute-dt");
    }
    
-   phiprof::start("getFieldsFromFsGrid");
-   // These should be done by initializeFieldPropagator() if the propagation is turned off.
-   volGrid.updateGhostCells();
-   technicalGrid.updateGhostCells();
-   getFieldsFromFsGrid(volGrid, BgBGrid, EGradPeGrid, technicalGrid, mpiGrid, cells);
-   phiprof::stop("getFieldsFromFsGrid");
-
    // Save restart data
    if (P::writeInitialState) {
       phiprof::start("write-initial-state");
-      phiprof::start("fsgrid-coupling-out");
```

Comment:
If a DRO for DCCRG BVOL exists, then that should be fetched from fsgrid before IO here, no?

Reply:
I'm not quite sure what you mean here. getFieldsFromFsGrid (the new gridGlue function) has been deleted from here because it's now called in initializeGrids on lin 445. If some DRO magic needs to be added here, then I could use some help with that.

Reply:
I only looked at the diffs, sorry. If at this stage (before the first half-step of leapfrogging was done) DCCRG already has all it needs (i.e. the initial state of fields) without any supplementary fetching, then we're good here. In the earlier implementation we had to fetch fsgrid fields here to have them ready for IO for initial-grid because they weren't yet put on dccrg otherwise.